bin/circleci/do-coverage

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+[ $CIRCLE_NODE_INDEX -eq 0 ] || exit 0
```

Comment:
Done. I think it would be a good idea to avoid the _code-in-YAML_ and move things to this kind of auxiliary scripts...